src/KeyworkerManagement/App.js

```diff
@@ -272,42 +252,31 @@ class App extends React.Component {
             />
             <Route
               exact
-              path="/manage-key-workers/key-worker-search/results"
+              path="/key-worker-search/results"
               render={() => (
                 <KeyworkerSearchResultsContainer displayBack={this.displayBack} handleError={this.handleError} />
               )}
             />
             <Route
               exact
-              path="/manage-key-workers/key-worker/:staffId"
+              path="/key-worker/:staffId"
```

Comment:
New-nomis-ui points to this url, and there may be other apps. Have you tracked them all or perhaps maybe redirects?

Reply:
I was aware of new-nomis-ui but not others... But a redirect solves my problem of release timings.  Thanks!